core/quill.js

```diff
@@ -467,6 +469,27 @@ Quill.events = Emitter.events;
 Quill.sources = Emitter.sources;
 // eslint-disable-next-line no-undef
 Quill.version = typeof QUILL_VERSION === 'undefined' ? 'dev' : QUILL_VERSION;
+Quill.LIST_STYLE_KEY = 'mso-list-data';
+
+Quill.replaceStyleAttribute = (html) => {
+  const tagAttrsRegex = /(?:(<[a-z0-9]+\s*))([\s\S]*?)(>|\/>)/gi;
```

Comment:
I see `[a-z0-9]+` for the content. Does it work for element without a content?

Reply:
this regex found all attributes in tag. If tag do not have attrs, it lead to that _tagAttrs_ will equal empty string and we do not wind any matches (we will not modify html in this case)